### PR TITLE
chore: Fix grammatical errors and touchup docs

### PR DESCRIPTION
**Pull request by salpalvv**
Mostly commas and slight rewording in markdown files.

I also bumped the README Cargo.toml to 0.3 due to the changes in documentation of `import!`

I tried not to be too opinionated on wording as a first pass.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,30 +2,33 @@
...
 
-If you are building with a nightly version of rust which supports workspaces you can use the `test-nightly.sh` script instead which puts the build artifacts in the root folder instead of in `c-api`.
+If you are building with a nightly version of rust which supports workspaces, you can use the `test-nightly.sh` script instead which puts the build artifacts in the root folder instead of in `c-api`.
 
 ## Pull requests
 
-Once you have made some changes you will need to file a pull request to get your changes merged into the main repository. If the code is still a work in progress it can still be a good idea to submit a PR as that will let other contributors see your progress and provide assistance (you may prefix the PR message with [WIP] to make it explicit that the PR is incomplete).
+Once you have made some changes, you will need to file a pull request to get your changes merged into the main repository. If the code is still a work in progress, it can still be a good idea to submit a PR. That will let other contributors see your progress and provide assistance (you may prefix the PR message with [WIP] to make it explicit that the PR is incomplete).
```
Comment (Marwes) [inline CONTRIBUTING.md]: > Once you have made some changes, you will need to file a pull request to get your changes merged into the main repository.

Does not read quite right to me with a comma.

File: README.md [hunk 2]
```diff
@@ -8,15 +8,15 @@ Gluon is a small, statically-typed, functional programming language designed for
 
 * **Statically typed** - Static typing makes it easier to write safe and efficient interfaces between gluon and the host application.
 
-* **Type inference** - Type inference ensures that types rarely have to be written explicitly giving the all the benefits of static types with none of the typing.
+* **Type inference** - Type inference ensures that types rarely have to be written explicitly giving all the benefits of static types with none of the typing.
```
Comment (Marwes) [inline README.md]: > written explicitly giving all

Maybe `written explicitly, giving all`?

**Comment by Marwes** [comment]
Looks good! You found a lot more wrong or questionable grammar than I would have guessed.

**Comment by salpalvv** [comment]
>Does not read quite right to me with a comma.

[See 2b](https://owl.english.purdue.edu/owl/resource/607/02/)

>After the test but before lunch, I went jogging.

There are commas I left out. I didn't want to do too much as I ended up touching a lot of statements.

**Pull request merged**